### ISSUE TITLE
Support various `Action` commands for server-client communication

### ISSUE THREAD

**Issue by jaeheonji** [zellij-org/zellij#1263]
### Overview

`Tmux` supports a wide range of Action Commands. And this can be classified into three main parts.
* Clients and Sessions
* Windows(Tabs) and Panes
* Key bindings

Each category has the following detailed command or purpose.
(The comparison was focused on the purpose rather than the details.)

#### Clients and Sessions

|         Purpose        |       Tmux       |                 Zellij                 |    Status   |
|:----------------------:|:----------------:|:--------------------------------------:|:-----------:|
|  attach target session | `attach-session` |                `attach`                |      OK     |
|    list all sessions   |  `list-sessions` |             `list-sessions`            |      OK     |
| destroy target session |  `kill-session`  | `kill-session` and `kill-all-sessions` |      OK     |
|  rename target session | `rename-session` |                  #998                  | In progress |
|   lock target session  |  `lock-session`  |                  None                  |     None    |
|  detach target client  |  `detach-client` |                  None                  |     None    |
|  switch target client  |  `switch-client` |                  None                  |     None    |

There is also a `start-server` command in `tmux`, but in `zelli`j, it can be executed through `--server` options. and it is also related to #1062

#### Windows(Tabs) and Panes

Currently there is no command related to this in `zellij`.  In `tmux`, Most of them have to do with move, resize, next and focus etc.
It is a work in progress in #413 and given the approach currently being implemented by @a-kenji, most commands appear to be compatible.

#### Key bindings

|               Purpose               |     Tmux    | Zellij | Status |
|:-----------------------------------:|:-----------:|:------:|:------:|
| send a key or keys to a window(tab) | `send-keys` |  None  |  None  |

tmux also has `send-prefix`, but this doesn't seem necessary for `zellij`.

If there is anything to be added or not necessary, please let me know 😃

**Comment by auronandace** [comment]
In case you are looking to update this issue:

For switching sessions there is #884
One feature I'd like to see is #809

**Comment by jaeheonji** [comment]
@auronandace Thank you! I updated it 👍

**Comment by a-kenji** [comment]
We can also send keys with #1265 through `Write: []`.

**Comment by uncomfyhalomacro** [comment]
> In case you are looking to update this issue:
> 
> For switching sessions there is #884 One feature I'd like to see is #809

Hello. Personal life has suddenly become too hectic for me so I put my Rust learning aside for quite a few months. Fortunately, a few weeks ago, I have free time now to review Rust and been started doing small projects with it. I will take a look again to #809 :smile:

**Comment by wcasanova** [comment]
bind C-a send-prefix
how can I replicate this in zellij
Bind C-a to send-prefix, so that C-a C-a sends a C-a to the application inside tmux/zellij
this is useful when you have a remote computer with zellij as well.

**Comment by jaeheonji** [comment]
@wcasanova

Hi!, Maybe I'm not understanding it well, but this seems to have something to do with keybindings.
Zellij has a tmux mode using C-b. You can change the settings related to this. Check out [default.yaml](https://github.com/zellij-org/zellij/blob/main/zellij-utils/assets/config/default.yaml). If it is something else, it would be helpful if you create a new issue.

**Comment by wcasanova** [comment]
Yes I have changed from C-b to C-a, but this is not the problem, it is when you have in your machine a zellij, and you have another remote machine inside the Zellij with another Zellij, I don't know if I understand.
The issue is to be able to send the key combination to the second Zellij, I do this with tmux.
